pyproject.toml

```diff
@@ -1,3 +1,43 @@
+[tool.poetry]
+name = "moabb"
+version = "0.2.1"
+description = "Mother of All BCI Benchmarks"
+authors = ["Alexandre Barachant", "Vinay Jayaram"]
+maintainers = ["Sylvain Chevallier <sylvain.chevallier@uvsq.fr>"]
+readme = "README.md"
+repository = "https://github.com/NeuroTechX/moabb"
+documentation = "http://moabb.neurotechx.com/docs/index.html"
+keywords = ["eeg", "datasets", "reproducibility", "bci", "benchmark"]
+license = "BSD-3-Clause"
+
+[tool.poetry.dependencies]
+python = "^3.6.1"
+mne = ">=0.19"
+pandas = "^1.0.0"
+h5py = "^3.0.0"
+scikit-learn = "^0.23"  # 0.24.0 breaks pyriemann, requires https://github.com/alexandrebarachant/pyRiemann/pull/93
+pyriemann = ">=0.2.6"
+matplotlib = "^3.0.0"
+seaborn = ">=0.9.0"
+PyYAML = "^5.0.0"
+coloredlogs = "^15.0"
+pyunpack = "^0.2.2"  # used only once, could be replaced with standard library call
+wfdb = "^3.3.0"
```

Comment:
As you said, we should get rid of it. Do you think it is possible to set it as an optional dependency? It is required only for one dataset.

Reply:
well...
Despite some negative comment on this package in dataset source - I've found no disadvantages in this package.

Also making this optionals requires some additional setup and since this tool is quite stable imo, I've skipped separating it to extra dependency.

If you are sure we don't need it in default package - I could change that of course

Reply:
The two issues with wfdb is that it requires numerous dependencies and that is has its own automated way to download data (that is quite slow for now). This is quite redundant with the code base that use functions defined in MNE to download, store and retrieve datasets (see https://github.com/NeuroTechX/moabb/blob/master/moabb/datasets/download.py). I think it is better if we could kept minimal dependencies to external packages for downloading dataset.  That being said, I agree that the `wfdb` package is maintained for now and that it is quite stable. We could keep it for now.

Reply:
okay, I'll label it as to be removed